extensions/EXT/SPV_EXT_image_raw10_raw12.asciidoc

```diff
@@ -0,0 +1,98 @@
+SPV_EXT_image_raw10_raw12
+=========================
+
+Name Strings
+------------
+
+SPV_EXT_image_raw10_raw12
+
+Contact
+-------
+
+To report problems with this extension, please open a new issue at:
+
+https://github.com/KhronosGroup/SPIRV-Headers
+
+Contributors
+------------
+
+- Kevin Petit, Arm Ltd.
+- Sven van Haastregt, Arm Ltd.
+
+Notice
+------
+
+Copyright (c) 2023 The Khronos Group Inc. Copyright terms at
+http://www.khronos.org/registry/speccopyright.html
+
+Status
+------
+
+Complete.
+
+Version
+-------
+
+[width="40%",cols="25,25"]
+|========================================
+| Last Modified Date | 2023-06-21
+| Revision           | 1
+|========================================
+
+Dependencies
+------------
+
+This extension is written against the SPIR-V Specification,
+Version 1.6 Revision 2.
+
+This extension requires SPIR-V 1.0.
+
+Overview
+--------
+
+This extension adds Image Channel Data Type definitions for RAW10 and RAW12
+image formats.
+
+
+Extension Name
+--------------
+
+To use this extension within a SPIR-V module, the following
+*OpExtension* must be present in the module:
+
+----
+OpExtension "SPV_EXT_image_raw10_raw12"
+----
+
+Modifications to the SPIR-V Specification, Version 1.6
+------------------------------------------------------
+
+Image Channel Data Type
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Modify Section 3.13, "Image Channel Data Type", adding these rows to the table:
+
+--
+[options="header"]
+|====
+2+^| Image Channel Data Type ^| Enabling Capabilities
+| 19 | *UnsignedIntRaw10EXT* | *Kernel*
+| 20 | *UnsignedIntRaw12EXT* | *Kernel*
```

Comment:
I was going to suggest that these new "image channel data types" shouldn't be guarded by the Kernel capability, but it looks like all of the other image channel data types are also guarded by the Kernel capability, as is the related [OpImageQueryFormat](https://registry.khronos.org/SPIR-V/specs/unified1/SPIRV.html#OpImageQueryFormat) instruction, so this is _probably_ OK as-is.

Adding a review comment for the record, and just in case somebody else has a similar reaction, but no action required.